eth/backend.go

```diff
@@ -651,16 +654,10 @@ func (s *Ethereum) StartMining(ctx context.Context, db kv.RwDB, mining *stagedsy
 		})
 	}
 
-	if s.chainConfig.ChainID.Uint64() > 10 {
+	if s.chainConfig.ChainID.Uint64() > 10 && s.initialCycle {
 		go func() {
-			skipCycleEvery := time.NewTicker(4 * time.Second)
-			defer skipCycleEvery.Stop()
-			for range skipCycleEvery.C {
-				select {
-				case s.sentriesClient.Hd.SkipCycleHack <- struct{}{}:
-				default:
-				}
-			}
+			time.Sleep(6 * time.Second)
```

Comment:
Why is loop replaced by firing it off once?

Reply:
I do it so after 6 seconds we skip the cycle in initial sync, since linkQueue is empty on initial sync I let it download some blocks for initial sync and if we dont have any blocks to process after 6 seconds I just skip the cycle